backend/src/bundles/users/user.controller.ts

```diff
@@ -170,6 +171,58 @@ class UserController extends BaseController {
         };
     }
 
+    /**
+     * @swagger
+     * /api/v1/users/update:
+     *   patch:
+     *     tags:
+     *       - Users
+     *     description: This endpoint updates a user
+     *     security:
+     *       - bearerAuth: []
+     *     requestBody:
+     *       required: true
+     *       content:
+     *         application/json:
+     *           schema:
+     *             type: object
+     *             properties:
+     *                fullName:
+     *                  type: string
+     *                username:
+     *                  type: string
+     *                dateOfBirth:
+     *                  type: string
+     *                weight:
+     *                  type: number
+     *                height:
+     *                  type: number
+     *                gender:
+     *                  type: string
+     *                  enum:
+     *                    - male
+     *                    - female
+     *                    - prefer not to say
```

Comment:
Shouldn't these fields be marked as nullable?

Reply:
Yes, missed that sorry.